src/TypeShape.SourceGenerator/TypeShapeIncrementalGenerator.cs

```diff
@@ -12,7 +12,11 @@ public sealed class TypeShapeIncrementalGenerator : IIncrementalGenerator
     public void Initialize(IncrementalGeneratorInitializationContext context)
     {
 #if LAUNCH_DEBUGGER
-        System.Diagnostics.Debugger.Launch();
+        if (!System.Diagnostics.Debugger.IsAttached)
```

Comment:
I had removed that guard initially because I found it didn't make any difference in practice. Did I miss something?

Reply:
I don't think so, I had some weird issues with debugging, so I copied some boilerplate from another srcgen in there, but apparently never reverted that change (probably because it's in #if). I'll remove that.